app/src/main/java/cs465/illinois/edu/dogonthequad/Dog.java

```diff
@@ -1,9 +1,25 @@
 package cs465.illinois.edu.dogonthequad;
 
+import java.lang.reflect.Array;
+import java.util.ArrayList;
 import java.util.UUID;
 
 public class Dog {
```

Comment:
Why are `Dog` and `User` being converted to classes with get functions?

Reply:
You'll also want to rebase this branch on develop, as some of these changes conflict with a branch I merged recently. (If you don't know what "rebase this branch on develop" means, that's totally fine. I can help with that.)

Reply:
Oh, do you just want the variables to be public?

Reply:
Yes, since that's how everything else is designed and, although most classes teach that all members should be private/protected, it's often more reasonable to just leave every member of a data transfer object public.